pkg/provider/aws/auth/auth.go

```diff
@@ -195,7 +195,7 @@ func NewGeneratorSession(ctx context.Context, auth esv1.AWSAuth, role, region st
 			return nil, err
 		}
 	}
-	config := aws.NewConfig()
+	awscfg, _ := config.LoadDefaultConfig(ctx)
```

Comment:
You removed `config` as a value and replaced it with config as an import. From there on config.Credential no longer exists.